pandas/tests/extension/test_arrow.py

```diff
@@ -1369,3 +1369,12 @@ def test_pickle_roundtrip(data):
 
     result_sliced = pickle.loads(sliced_pickled)
     tm.assert_series_equal(result_sliced, expected_sliced)
+
+
+def test_astype_from_non_pyarrow(data):
+    # GH49795
+    pd_array = data._data.to_pandas().array
```

Comment:
assert that this is a TimedeltaArray?

Reply:
This test is not timedelta specific. It tests all of the types in the `data` fixture. Timedelta was the only one that raised with `{non-pyarrow array}.astype({ArrowDtype})`.